README.md

```diff
@@ -124,6 +125,7 @@ something that can be coerced into a `URI` such as a `String`) with three option
 * `:logger`: a [`Logger`](http://ruby-doc.org/stdlib/libdoc/logger/rdoc/Logger.html)-compatible `logger`, defaults to a platform-specific null logger;
 * `:timeout`: a `Numeric` number of seconds to wait before timing out any HTTP requests, defaults to 60;
 * `:bearer_token`: a `String` bearer token to authorize any HTTP requests, defaults to `nil`.
+* `:headers`: a `Hash` containing custom HTTP headers, defaults to `empty hash`.
```

Comment:
Would it be clearer to say it defaults to `{}` or drop the backticks altogether?